build.gradle

```diff
@@ -55,7 +55,7 @@ allprojects {
   }
 
   dependencies {
-    compile group: 'net.logstash.logback', name: 'logstash-logback-encoder', version: '5.3'
+    compile group: 'net.logstash.logback', name: 'logstash-logback-encoder', version: '6.0'
```

Comment:
hmm do we still need this?

Reply:
not entirely sure. i've seen json logs on frontend only or was it backend as well while investigating one of the processes few weeks ago 🤔

Reply:
we should probably bin it i think

Reply:
👍. will leave for separate pr. there are unit/integration tests. after removal library will become super tiny. not sure how much `java-logging-(httpcomponents|spring)` are used out there either

Reply:
not that useful really, they do log the request timing. 
All that info goes into app insights but it may be useful to have it in logs too...